comp_sci_gender_bias/pipeline/text_data/parse_bit_data.py

```diff
@@ -0,0 +1,43 @@
+# Parse BIT raw data
+
+import logging
+import json
+import os
+import docx
+
+from comp_sci_gender_bias import PROJECT_DIR
+
+DATA_DIR = f"{PROJECT_DIR}/inputs/data/"
+BIT_DIRS = ["bit_compsci_descriptions", "bit_geo_descriptions"]
+
+
+def getText(filename: str) -> str:
+    """Reads and parses a doc file from BIT"""
+
+    doc = docx.Document(filename)
+    fullText = []
```

Comment:
Variable `fullText` should be in camel case: `full_text` as per the style guidelines